substrate/client/network/src/litep2p/mod.rs

```diff
@@ -702,12 +702,12 @@ impl<B: BlockT + 'static, H: ExHashT> NetworkBackend<B, H> for Litep2pNetworkBac
 					None => return,
 					Some(command) => match command {
 						NetworkServiceCommand::GetValue{ key } => {
-							let query_id = self.discovery.get_value(key.clone()).await;
-							self.pending_get_values.insert(query_id, (key, Instant::now()));
+							let query_id = self.discovery.get_value(key.clone().to_vec().into()).await;
```

Comment:
```suggestion
							let query_id = self.discovery.get_value(key.to_vec().into()).await;
```

No need to clone it first here and everywhere else.